src/main/kotlin/io/github/bbortt/buessle/app/web/CreateGameController.kt

```diff
@@ -15,15 +15,16 @@ class CreateGameController {
     @PostMapping("/validate")
     // TODO: Validate room by uuid
     // Return name if exists, boolean if session is owner (in case he disconnects)
-    fun validateRoom(@RequestBody validateRoom: ValidateRoom) = ValidatedRoom("random name", true);
+    fun validateRoom(@RequestBody validateRoom: ValidateRoom) = ValidatedRoom("random name", 1);
```

Comment:
The idea is to associate the session and username to a new player ID and return this. Or return the existing ID if it already exists with this session.
The player who created the game should be associated with ID "0".